README.md

```diff
@@ -94,6 +94,7 @@ To lint the project, run:
 ```cmd
 golangci-lint run
 ```
+> Note: `golangci-lint` seems to panic on M1 machines with a `can't load fmt` error. This issue is documented [here](https://github.com/golangci/golangci-lint/issues/2374). While this is still being fixed, it seems that installing `golangci-lint` from source resolves the issue. 
```

Comment:
oh I meant to ask about this on the other PR but:

what does "seem to panic" mean?  is it flaky?

Reply:
Not the best wording here... Meant to say that it panics and it seems like it's an M1 issue. I'll clarify this.